### PR TITLE
 store: use uniform log format for store package (#9664)

### PR DESCRIPTION
**Pull request by jackysp**
### What problem does this PR solve? 
Cherry-pick #9664 to release 2.1.

### CODE REVIEW

File: store/store.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
+// Copyright 2018 PingCAP, Inc.
```
Comment (zimulala) [inline store/store.go]: s/2018/2019?

File: store/tikv/2pc.go [hunk 2]
```diff
@@ -138,14 +139,23 @@ func newTwoPhaseCommitter(txn *tikvTxn, connID uint64) (*twoPhaseCommitter, erro
 	const logSize = 4 * 1024 * 1024 // 4MB
 	if len(keys) > logEntryCount || size > logSize {
 		tableID := tablecodec.DecodeTableID(keys[0])
-		log.Infof("[BIG_TXN] con:%d table id:%d size:%d, keys:%d, puts:%d, dels:%d, locks:%d, startTS:%d",
-			connID, tableID, size, len(keys), putCnt, delCnt, lockCnt, txn.startTS)
+		logutil.Logger(context.Background()).Info("[BIG_TXN]",
+			zap.Uint64("con", connID),
```
Comment (zimulala) [inline store/tikv/2pc.go]: s/con/conn

File: store/store.go [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+// New creates a kv Storage with path.
+//
+// The path must be a URL format 'engine://path?params' like the one for
+// session.Open() but with the dbname cut off.
+// Examples:
+//    goleveldb://relative/path
+//    boltdb:///absolute/path
+//
+// The engine should be registered before creating storage.
+func New(path string) (kv.Storage, error) {
```
Comment (zimulala) [inline store/store.go]: Do we use it?

**Comment by jackysp** [comment]
/run-all-tests tidb-test=release-2.1 tikv=release-2.1 pd=release-2.1

**Comment by jackysp** [comment]
PTAL @zimulala

**Pull request merged**